lang/cs.json

```diff
@@ -0,0 +1,55 @@
+{
+    "VISUAL_ACTIVE_EFFECTS": {
+        "SETTINGS": {
+            "ICON_SIZE": {
+                "NAME": "Velikost ikony",
+                "HINT": "Změte velikost ikony v pixelech. Výchozí: 50."
+            },
+            "DAYS_PER_WEEK": {
+                "NAME": "Dnů v týdnu",
+                "HINT": "Počet dnů v týdnu."
+            },
+            "WEEKS_PER_MONTH": {
+                "NAME": "Týdnů v měsíci",
+                "HINT": "Počet týdnů v měsíci."
+            },
+            "MONTHS_PER_YEAR": {
+                "NAME": "Měsíců v roce",
+                "HINT": "Počet měsíců v roce."
+            },
+            "EXTRA_DAYS_PER_YEAR": {
+                "NAME": "Navíc dnů v roce",
+                "HINT": "Počet dnů navíc v roce."
+            }
+        },
+        "LABELS": {
+            "TEMPORARY": "Dočasné",
+            "DISABLED": "Zakázané",
+            "DETAILS": "Detaily"
+        },
```

Comment:
Missing localization was fixed. This can be added here (see the en.json).

Reply:
Added "PASSIVE"